### PR TITLE
Fix names and add pdfs for presentations

### PR DESCRIPTION
**Pull request by RossTate**
I was going through presentations with some students and realized the pdfs are previewable on GitHub but the powerpoints are not, so I added them and fixed up a few names and links while I was at it.

### CODE REVIEW

File: main/2020/CG-02.md [hunk 1]
```diff
@@ -980,6 +980,7 @@ RT: What's the indirection you're referring to.
 BT: So suppose you have class A, some java library two fields. User library imports the type with one field. Now you compile the user module offline, it has an extra indirection, it doesn't know where the field is at. A table of numbers that you have to lookup through, another load.
 
 RT: Another way to do it is talking to other communities: two other ways 
+RT: Another way to do it is talking to other communities: two other ways
```
Comment (binji) [inline main/2020/CG-02.md:983]: duplicated line

File: main/2020/CG-04-14.md [hunk 2]
```diff
@@ -39,4 +39,4 @@ Installation is required, see the calendar invite.
 
 ## Meeting Notes
 
-To be added after the meeting.
+Ross Tate presenting slides for Stack and Control Primitives [(pdf)](presentations/2020-04-14-tate-stack-primitives.pdf) [(pptx)](presentations/2020-04-14-tate-stack-primitives.pptx)
```
Comment (binji) [inline main/2020/CG-04-14.md:42]: Interesting, I wonder what happened to the notes for this meeting...? I'll check.

**Review by binji — APPROVED** [review]

**Comment by binji** [comment]
Thanks for adding the PDFs, agree it's nicer to read directly in GitHub!

File: main/2020/CG-02.md (hunk 1 above)
Comment (RossTate) [reply main/2020/CG-02.md]: Oops, yeah, my editor was misconfigured and I noticed after the fact that it had stripped all the white space. I thought I undid all the unwanted changes, but I guess I missed half of one, heheh. Fixed in the forced push.

**Pull request merged**